cypress.json

```diff
@@ -0,0 +1,39 @@
+{
+	"baseUrl": "http://localhost:5001",
+	"ignoreTestFiles": [
+	"**/__snapshots__/**",
+	"**/__image_snapshots__/**"
+	],
+	"env": {
```

Comment:
Looks to me like this is default copy/paste from [Github](https://github.com/meinaart/cypress-plugin-snapshots). These are an option if we want to configure anything.